WcaOnRails/app/models/competition_big_json.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class CompetitionBigJson < ApplicationRecord
+  belongs_to :competition, required: true
```

Comment:
> Many competitions won't have any CCIF data, and that seems fine.

Sure, but every CCIF data should have a competition, and that's what we're talking about here, right?